kolyanTask/task.js

```diff
@@ -0,0 +1,62 @@
+const AIRCRAFT_TYPES = {
+    type1: `plane`,
+    type2: `copter`,
+}
+
+class AircraftTypeError extends Error {
+    constructor(message) {
+        super(message);
+        this.name = 'AircraftTypeError';
+    }
+}
+
+
+
+class Aircraft {
+    constructor() {}
+    
+    type = null;
+
+    setType(type) {
+        let arrOfTypes = Object.values(AIRCRAFT_TYPES)
+
+        if (!arrOfTypes.includes(type)) {
+            throw new AircraftTypeError(`Wrong aircraft type!`)
+        } 
+
+        this.type = type; 
+    }
+
+    motors = 0;
+
+    addMotors(count) {
+        if (this.type == null) {
+            throw new AircraftTypeError('Aircraft type not set!')
+        }
+        if (this.type === AIRCRAFT_TYPES.type1) {
+            if (count <= 2) {
+                this.motors = count;
+            } else  {
```

Comment:
```suggestion
                this.motors = count;
                return;
            } else  {
```

in this case you don't need to check (count > 3) in next if (and the whole next `if` won't be executed)